Abbyy.CloudSdk.V2.Client.Sample/Program.cs

```diff
@@ -32,35 +37,105 @@ public class Program
 		/// </summary>
 		private const string ServiceUrl = "https://cloud-eu.ocrsdk.com";
 
-		private static AuthInfo _authInfo;
+		private static int _retryCount = 3;
+		private static int _delayBetweenRetriesInSeconds = 3;
+		private static string _httpClientName = "OCR_HTTP_CLIENT";
+
+		private static readonly AuthInfo AuthInfo = new AuthInfo
+		{
+			Host = ServiceUrl,
+			ApplicationId = ApplicationId,
+			Password = Password
+		};
+
+		private static ServiceProvider _serviceProvider;
+		private static HttpClient _httpClient;
 
 		public static async Task Main()
 		{
-			// Init library
-			_authInfo = new AuthInfo
+			// Init client
+			// You could also call GetOcrClient to use client without retry policy
+			using (var ocrClient = GetOcrClientWithRetryPolicy())
```

Comment:
```
// You could also call GetOcrClient to use client without retry policy
using (var ocrClient = GetOcrClientWithRetryPolicy())
```
1) comment and code in using(...) block are opposite in meaning

Reply:
But that's the point of the comment, it notes that you can create OcrClient without retry policy

Reply:
I see. incorrectly interpreted this comment and code below